Documentation/tutorial/intro markdown lite.md

```diff
@@ -0,0 +1,183 @@
+# Markdown Lite
+
+## Introduction
+
+Markdown lite is a simple markdown tool to markup `md` file.
+
+## Design goal
+
+We write this tool for good extensiblity, so our implements should obay following order:
```

Comment:
"extensibility", "implementation" and "obey"?

---
In reply to: [68715081](https://github.com/dotnet/docfx/pull/436#discussion_r68715081) [](ancestors = 68715081)